virttest/qemu_devices/qcontainer.py

```diff
@@ -2896,6 +2896,21 @@ def throttle_group_define_by_params(self, group_params, name):
         props = json.loads(group_params.get("throttle_group_parameters", "{}"))
         return QThrottleGroup(name, props)
 
+    def thread_context_define_by_params(self, params, name):
+        """
+        Create thread-context object from params.
+        """
+        tc_params = Params()
+        suffix = '_thread_context'
```

Comment:
ok, we'd suggest using prefix instead because the suffix was occupied for representing the object id of the logical devices, otherwise there might be some issues upon corner cases.

Reply:
@luckyh updated now, thanks!